Knob width can be calculated as a percentage of window width

oliviaboni opened the pull request:
Hi @joshjg, I'm using this knob in an app I'm working on and I needed it to be responsive. Here's a quick solution I came up, where, if the width is 1 or less, it is taken as a percentage of window.innerWidth. Works for me. Great resource! Thanks

Commit:
Knob width can be calculated as a percentage of window width

JulianAnthes commented:
Hey @oliviaboni, [I've also made a pull request, to update the knobs dimensions, when it's needed.](https://github.com/joshjg/react-canvas-knob/pull/12)

Maybe this could help you, too. 🥂

joshjg commented:
@oliviaboni  @dschu-lab Sorry for the wait. Merged @dschu-lab's PR. Will keep this open for now - let me know if you think this is still necessary.

Commit:
Allow for two cursors

Commit:
Update to React 16, incorporate upstream changes

Commit:
Update build to not rely on hot reloader to be present

This is needed for prod deployments to work.

Also tidyed up the build and use prepublish to compile when needed.

oliviaboni committed:
Merge pull request #1 from kellycampbell/hot-reload-fix

Update build to not rely on hot reloader to be present

Commit:
Fix touchEnd event and update package.json

JulianAnthes commented on package.json:
```diff
@@ -1,14 +1,15 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "repository": "git@github.com:joshjg/react-canvas-knob.git",
+  "repository": "git@github.com:oliviaboni/react-canvas-knob.git",
```
Since you're going to do a merge request to @joshjg, we should leave this line as it is. 😅

Commit:
Allow more robust modifications to cursors
